### PR TITLE
Include incompletely tagged files

### LINKED ISSUE #4: Properly support incompletely tagged files
> Also
> ```
> Kein Artist
> Untagged/
> 
> Kein Album
> Artist/Untagged
> 
> Kein Titel
> Artist/Album/File name
> ```

### PR DESCRIPTION
**Pull request by deingithub**
Fixes #4

### CODE REVIEW

File: src/indexing.c [hunk 1]
```diff
@@ -17,6 +17,8 @@
 #include "mufs.h"
 #include "db.h"
 
+static char *untagged = "<Untagged>";
```
Comment (nunner) [inline src/indexing.c]: <Untagged> may interfere with file paths and looks a bit unpleasant, just "Untagged" would be better

File: src/indexing.c [hunk 2]
```diff
@@ -30,8 +32,17 @@ get_tags(const char *fpath)
 
         file->path = fpath;
         file->tags->artist = taglib_tag_artist(tag);
+        if (strlen(file->tags->artist) == 0)
```
Comment (nunner) [inline src/indexing.c:35]: how about we do something like `taglib_tag_artist(tag) ? taglib_tag_artist(tag) : untagged`?

**Review by nunner — COMMENTED** [review]
I was just working on this issue :D. See my review for some comments

File: src/indexing.c (hunk 2 above)
Comment (deingithub) [reply src/indexing.c:35]: Would that actually work? It's not a null pointer, just an empty string.

File: src/indexing.c (hunk 1 above)
Comment (deingithub) [reply src/indexing.c]: Sure.

File: src/indexing.c (hunk 2 above)
Comment (nunner) [reply src/indexing.c:35]: Right, we could use [0], but at this point, the code would look about the same

**Comment by nunner** [comment]
also, I think the checks you added in #1 are unnecessary now, no?

**Comment by deingithub** [comment]
Indeed

**Pull request merged**